dotCMS/src/main/java/com/dotcms/publishing/output/DirectoryBundleOutput.java

```diff
@@ -0,0 +1,111 @@
+package com.dotcms.publishing.output;
+
+import com.dotcms.publishing.BundlerUtil;
+import com.dotcms.publishing.PublisherConfig;
+import com.google.common.annotations.VisibleForTesting;
+import com.liferay.util.FileUtil;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.util.Collection;
+
+public class DirectoryBundleOutput extends BundleOutput {
```

Comment:
doc it

Reply:
done https://github.com/dotCMS/core/pull/20173/commits/e491040a5cc382fa8a25457ba027679b4afa86b7#diff-f2555462ad9b90e9c92d2e27a63788c8a8b50bd6242099982cf202919481710dR16